env/elasticsearch/Dockerfile

```diff
@@ -1,5 +1,5 @@
 #https://www.docker.elastic.co/
-FROM docker.elastic.co/elasticsearch/elasticsearch:6.5.2
+FROM docker.elastic.co/elasticsearch/elasticsearch:7.0.0-alpha1
```

Comment:
Lets use `docker.elastic.co/elasticsearch/elasticsearch:7.0.0-alpha1-SNAPSHOT` during the development for this. It will make the CI more flaky here as it could be that new breaking changes just came in but lots of changes have happened since `7.0.0-alpha1`.

Reply:
@ruflin I'm facing this issue with the snapshot, and I didn't find documentation about it :

```
bin/elasticsearch-plugin install --batch ingest-attachment
ERROR: attempted to install release build of official plugin on snapshot build of Elasticsearch
```

Do you have an idea about it ?

Reply:
Perhaps the line here helps (different plugin, but same mechanism should work): https://github.com/elastic/beats/blob/6.4/testing/environments/docker/elasticsearch/Dockerfile#L39

Reply:
yes it's useful but I'm managing to find the right URL :)

this is not working
```
https://snapshots.elastic.co/downloads/elasticsearch-plugins/ingest-attachment/ingest-attachment-7.0.0-alpha1-SNAPSHOT.zip```

any hint on this ?

Reply:
@ruflin :) finally I've been able to build the URL 👍 

```https://snapshots.elastic.co/downloads/elasticsearch-plugins/ingest-attachment/ingest-attachment-7.0.0-SNAPSHOT.zip```